mk/uvmt/vsim.mk

```diff
@@ -177,8 +176,25 @@ VSIM_UVM_ARGS      = +incdir+$(UVM_HOME)/src $(UVM_HOME)/src/uvm_pkg.sv
 #  endif
 #endif
 
-
-ifeq ($(call IS_YES,$(USE_ISS)),YES)
+GCC_PATH			= $(HOME)/opt/gcc-11.4.0/bin/gcc
+
+VSIM_FLAGS        	+= -dpicpppath $(GCC_PATH)
+VSIM_FLAGS 			+= -noautoldlibpath
+
+# This must currently always be exported, since the reference_model_wrap is used in uvmt_cv32e40s_tb.sv
+# TODO: Make a "dummy" reference model, that is loaded when it is disabled, like ImperasDV. 
+# Or find another solution to handle swapping different ISSs
+export FILE_LIST_RM      ?= -f $(DV_UVMT_PATH)/reference_model.flist
+
```

Comment:
Since ImperasDV support is fairly hard-coded, "disabling" IDV and enabling the RM this way is not optimal. Suggestions on how this can be done is very welcome

Reply:
> enabling the RM this way is not optimal

Could you point me specifically to what way you mean?

Do you mean setting args on the cli to say no to one and yes to the other? Or are you referring to the defines used in the systemverilog code?

Reply:
Now, USE_ISS is synonymous with ImperasDV, which is a bit unspecific. This could instead be USE_IDV, or different ISSs could be chosen with ISS=IMPERASDV or ISS=RM, etc. Additionally, the name "ISS" is not that accurate either since both of these are more than an ISS. However, is this probably not within the scope of this PR?

Reply:
I like the idea of keeping `USE_ISS`, and to let some additional variable default to `IDV` but be overridable to e.g. `RM`.
Because then the old behavior will be preserved (no breakage), and the new behavior will be possible to enable.s

Reply:
This functionality was added with the merge from master